pages/linux/entr.md

```diff
@@ -0,0 +1,23 @@
+# entr
+
+> Run arbitrary commands when files change.
+
+- Rebuild with `make` if any files in any subfolders change:
+
+`ag -l | entr make`
+
+- Rebuild and test with `make` if `.c` sources in current directory changes:
+
+`ls *.c | entr 'make && make test'`
+
+- Reload the MuPDF viewer whenever the source `.pdf` changes:
+
+`ls *.pdf | entr pkill -HUP mupdf`
+
+- Reload the spawned childprocess with `SIGTERM` before restarting:
```

Comment:
Perhaps `subprocess` or `child process` would read better here?

Also, what exactly do you mean by "reload"?

Reply:
Is this more clear?
```suggestion
- The reload (`-r`) flag sends a `SIGTERM` to any previously spawned ruby subprocesses before executing `ruby main.rb`:
```

Reply:
I think so! Perhaps we could trim it back to this?

```
Send a `SIGTERM` to any previously spawned ruby subprocesses before executing `ruby main.rb`
```

Reply:
Definitely! I've updated the commit.